Dragon-Prince-fanmade/sheet.json

```diff
@@ -0,0 +1,10 @@
+{
+  "html": "dragon-prince.html",
+  "css": "dragon-prince.css",
+  "authors": "Senjak",
+  "roll20userid": "5779412",
+  "preview": "Dragon-Prince.png",
+  "instructions": "For use with the Cortex Prime Tales of Xadia.",
+  "legacy": false,
+  "instructions": "Here be dragons..."
```

Comment:
might be best to only have one "instructions" sections, having two might cause troubles.